examples/jenkins/pipeline/pipelinetemplate.json

```diff
@@ -19,6 +19,9 @@
         "creationTimestamp": null,
         "labels": {
           "name": "sample-pipeline"
+        },
+        "annotations": {
+          "downstream.openshift.io/deployments": "[{\"name\": \"frontend\", \"namespace\": \"\"}]"
```

Comment:
(and yeah, the DC isn't actually created by the pipeline, but the deployment that came from the DC is)